content/html/ibm-z-request-ci.html

```diff
@@ -0,0 +1,141 @@
+<div id="content">
+<!-- Formsender error script -->
+<script src="../../../theme/js/formsender-error.js"></script>
+  <div class="field field-name-body field-type-text-with-summary field-label-hidden">
+    <div class="field-items">
+      <div class="field-item even" property="content:encoded">
+        <p>Please use the form below to request access to the IBM Z CI service provided by the OSUOSL.</p> <p>This
+        access is intended only for <b><i>free and open source</i></b> projects who qualify and are approved by
+        the OSUOSL.
+        <!-- TODO: For proprietary sourced projects or products please use other resources which can be
```

Comment:
@ramereth Are there any equivalent links to IBM Z resources (for proprietary projects) we might like to include?

Reply:
No, but they do have this [link](https://www.ibm.com/developerworks/community/groups/community/lozopensource) to their community. Please add here and the Z landing page too.